pipelines/push-snapshot-to-quay/push-snapshot-to-quay.yaml

```diff
@@ -26,6 +26,14 @@ spec:
     - name: taskGitRevision
       type: string
       description: The revision in the taskGitUrl repo to be used
+    - name: release
+      description: Namespaced name of release - should be in format "namespace/name"
+      type: string
+      default: ""
```

Comment:
Should I provide a default here? If not, then I can get rid of some of the error checking in the task.

Reply:
I removed this default to be consistent with the releaseplan.